packages/@sanity/base/src/change-indicators/overlay/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface ConnectorLinePoint {
```

Comment:
Can't we put types in `.d.ts` files and that way get to use them in the project without having to import them? :)

Reply:
Hmm... I think the same reasons for preferring explicit imports over implicit globals applies to types as much as it does for values?

Reply:
I just didn't know preferring explicit imports of types was a thing - but it makes sense :) No worries!